projects/packages/forms/src/contact-form/js/date-picker.js

```diff
@@ -0,0 +1,12 @@
+import domReady from '@wordpress/dom-ready';
+import Datepicker from 'vanillajs-datepicker/Datepicker';
+
+import 'vanillajs-datepicker/css/datepicker.css';
```

Comment:
should we use this one instead? https://developer.wordpress.org/block-editor/reference-guides/components/date-time/

Reply:
I'd rather not because we would need to ship React on frontend, which is a really heavy library for something as trivial as this.

On core they're also leaning towards using browser's native date picker (internal ref p1736937404679249-slack-CNGQYA3B9)

Reply:
nice, I like the idea of using browser's native date picker

Reply:
The only worry I have is how it looks and how themeable it is by users. This JS one look really neat for example.